### DISCUSSION TITLE
Conditional rules don't work on field group element type

### DISCUSSION THREAD

**Discussion by samuel-clara** [solspace/craft-freeform#1478] [Questions & Answers]
### What happened?

Conditional rules don't work correctly on field group element type on a multipage form.

### How can we reproduce this?

1. Add a group element on page 3
2. Create a rule to show this group element based on the value of a dropdown field on page 2
3. Group field is shown regardless of the dropdown field value

### Environment

- Freeform Pro 5.5.5
- Craft 5.2.9

**Comment by kjmartens** [comment]
Sorry for the delay @samuel-clara,

I tried this on my test site and I can't reproduce the issue. It seems to correctly respect the conditional rules.

Did you ever get this working? 🙂 

If the issue still exists, please let me know.
